openlibrary/templates/type/author/edit.html

```diff
@@ -109,7 +111,7 @@ <h1>$_("Edit Author")</h1>
                         <div class="label">Identifiers</div>
```

Comment:
Question mark icon should appear to the right of the "Identifiers" label.  It appears beneath the form inputs now:
![question_icon_before](https://user-images.githubusercontent.com/28732543/135158899-46958dd3-77a8-47d7-9aa3-d63e4c7c1ad0.png)